compiler-rt/test/ubsan/TestCases/TypeCheck/misaligned.cpp

```diff
@@ -47,6 +50,16 @@ int main(int, char **argv) {
     return *p && 0;
     // CHECK-STACK-LOAD: #0 {{.*}}main{{.*}}misaligned.cpp
 
+  case 'L': {
+    int x;
+    // CHECK-MEMCPY-LOAD: misaligned.cpp:[[#@LINE+4]]{{(:16)?}}: runtime error: load of misaligned address [[PTR:0x[0-9a-f]*]] for type 'const void *', which requires 4 byte alignment
```

Comment:
Actually in this case I'd rather ask, why compiler believe that "p" suppose to be aligned?

Reply:
Never mind, I see.